tools.yaml

```diff
@@ -2117,6 +2117,14 @@
   categories:
   - analysis
   - proprietary
+- name: RunSafe C/C++ Build-time SBOM Generator
+  publisher: RunSafe Security
+  description: RunSafe Security generates a C/C++ SBOM at build-time without the need for a package manager. RunSafe works flexibly across build environments, reports on all library dependencies, and is enriched with information about the final target.
+  websiteUrl: https://runsafesecurity.com/c-c-plus-plus-sbom/?utm_campaign=9722897-Software%20Tools&utm_source=Website&utm_medium=CycloneDX&utm_content=SBOM
+  categories:
+  - proprietary
+  - build-integration
+  - github-action
```

Comment:
the `github-action` category describes as follows
https://github.com/CycloneDX/tool-center/blob/ca3ac6ca0786e25402c3f64d8bea6ac887cddb02/tool-categories.yaml#L19-L21

so you actually provide a github action  for that proprietary tooling?

Reply:
sounds good.